app/Contracts/DivisionApiContract.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+namespace App\Contracts;
+
+use App\Models\Area;
+use App\Models\User;
+
+interface DivisionApiContract
```

Comment:
I like a good contract!

Reply:
Indeed! Out of scope, but this is where the TaskContract should perhaps live as well